irma-css/src/components/irma-form.scss

```diff
@@ -72,6 +70,59 @@
   border: 1px solid $border-color;
   border-radius: $border-radius;
 
+  .irma-web-content {
+    @include reset;
+    @include center;
+    flex: 1 1 auto;
+    margin: 0 1rem 1rem 1rem;
+
+    background-color: $content-bg-color;
+    border-radius: $border-radius * 0.6;
+
+    .irma-web-centered {
+      @include reset;
+      height: auto;
+      text-align: center;
+
+      img,
+      canvas {
+        cursor: none;
+      }
+    }
+
+    p {
+      @include reset;
+      @include font($size: 15px);
+      margin: 1em 4em;
+
+      @include on-small-screen {
+        margin: 1em 2em;
+      }
+    }
+
+    // Make sure margin-top of first child and margin-bottom of last child
+    // are equal to prevent issues while centering.
+    > * {
+      &:first-child {
+        margin-top: 0;
+      }
+      &:last-child {
+        margin-bottom: 0;
+      }
+    }
+
+    a {
+      @include reset;
+      @include font($color: $content-link-color, $size: 15px);
+      text-decoration: underline;
+      cursor: pointer;
+
+      &:hover {
+        color: $content-link-color-hover;
+      }
+    }
+  }
+
```

Comment:
In CSS less specific rules should precede more specific rules. Therefore the linter required me to switch both blocks from position.